app/components/lock_old_posts.py

```diff
@@ -20,17 +23,21 @@ async def check_for_old_posts(message: discord.Message) -> None:
         or not post_is_solved(post)
         or (
             (starter := post.starter_message) is not None
-            and starter.created_at > now - dt.timedelta(days=30)
+            # Do a quick check for the post starter message, whose timestamp is
+            # readily available without fetching messages from Discord. This
+            # works around ratelimits due to a high volume of fetches when
+            # messages are sent in the #help channel.
```

Comment:
I'm not sure I like the wording—the comment seems to imply (to me) that this is _only_ a workaround, while in reality it's an early guard that helps narrow the relevant messages down and was just overlooked.

Reply:
True, `works around` for something that isn't just a work around isn't particularly good wording.  Does this seem okay?
```suggestion
            # Also check the post starter message, whose timestamp is
            # readily available without fetching messages from Discord. This
            # reduces ratelimits due to a high volume of fetches when
            # messages are sent in the #help channel.
```

Reply:
Yeah that looks okay